### DISCUSSION TITLE
SubscribeToShard for aws_sdk_kinesis?

### DISCUSSION THREAD

**Discussion by avandesa-fw** [awslabs/aws-sdk-rust#638] [General]
Hello! From what I can tell, the kinesis crate doesn't appear to support the `SubscribeToShard` operation? This prevents consumers from taking advantage of the [dedicated throughput/enhanced fan-out](https://docs.aws.amazon.com/streams/latest/dev/enhanced-consumers.html). Are there any plans to implement this, or am I missing something in the docs? `RegisterStreamConsumer` is implemented and is required for `SubscribeToShard`, but I don't see the subscription method itself.

https://docs.rs/aws-sdk-kinesis/latest/aws_sdk_kinesis/struct.Client.html

Just wanted to double check I wasn't missing anything before opening a feature request. Thanks!

**Comment by Velfi** [comment]
We actually already have a feature request for this https://github.com/awslabs/aws-sdk-rust/issues/213
We do plan to implement it at some point but I don't think it'll happen soon.

**Reply by avandesa-fw** [reply]
Ah, guess I missed that. Thanks for the info!
